test/core/templates/gif-module-test.js

```diff
@@ -0,0 +1,33 @@
+target = 'test_outputs';
+
+/**
+ * @method moduleTest
+ * @description a common test for modules
```

Comment:
Please start every comment with a capital letter and end with a `.`.

Reply:
done